sass/base/_css_variables.scss

```diff
@@ -62,6 +62,7 @@
     --denim-status-online-rgb: 61, 184, 135;
     --denim-button-bg-rgb: 28, 88, 217;
     --center-channel-text-rgb: 63, 67, 80;
+    --title-color-indigo-500-rgb: 30, 50, 92;
```

Comment:
i havent checked but there is already `--title-color-indigo-500` ? 🤔

Reply:
Yeah, it's the same colour just written in rbg form. In the mocks, it was using the`--title-color-indigo-500` colour at a lower opacity. I saw that in same other areas of the code, when it involved opacity, it used the rgb value of the colour and passed in a 4th parameter for the opacity. So I tried to follow a similar format, but `--title-color-indigo-500` in rgb form didn't exist before. So I added it there.

Reply:
yes sounds good thanks